daemon/src/peer/include.rs

```diff
@@ -15,7 +15,7 @@ where
     S: Clone + Signer,
 {
     match state::update_include(&peer, urn.clone()).await {
```

Comment:
Could change this to to `if let Err(err)`

Reply:
Fixed in e0f8ba665088506a52784ac5335cbb639d40818b